R/setThreadCount.r

```diff
@@ -0,0 +1,62 @@
+#' Function to set the thread count for a specific \code{data.table} function
+#'
+#' @param benchmarkData A \code{data.table} of class \code{data_table_threads_benchmark} containing benchmarked timings with corresponding thread counts.
+#'
+#' @param funcName The name of the \code{data.table} function for which to set the thread count.
+#' 
+#' @param verbose Option (logical) to enable or disable detailed message printing.
+#'
+#' @return NULL.
+#'
+#' @details Sets the thread count to either the optimal (fastest median runtime) or recommended value (default) based on the chosen type argument for the specified \code{data.table} function based on the results obtained from \code{findOptimalThreadCount()}.
+#'
+#' @export
+#'
+#' @import data.table
+#'
+#' @examples
+#' \dontrun{
+#' # Finding the best performing thread count for each benchmarked data.table function with a data size of 10000000 rows and 10 columns:
+#' benchmarkData <- data.table.threads::findOptimalThreadCount(1e7, 10)
+#' # Setting the optimal thread count for the 'forder' function:
+#' setThreadCount(benchmarkData, "forder", "optimal")
+#' # Can verify by checking benchmarkData and getDTthreads()
+#' # Setting the recommended thread count for the 'nafill' function:
+#' setThreadCount(benchmarkData, "nafill", "recommended")
+#' }
+
+setThreadCount <- function(benchmarkData, functionName, type = "recommended", verbose = FALSE)
```

Comment:
it seems this function is missing an argument for the efficiency factor (default 0.5) for recommended